src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,143 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.customexception.ValidationExceptionIncorrectValue;
+import core.basesyntax.customexception.ValidationExceptionNullValue;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User testUser;
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setDefaultData() {
+        testUser = new User();
+        Storage.people.clear();
+    }
+
+    @Test
+    void containsUserCorrectData_OK() {
+        testUser.setLogin("CorrectLogin");
+        testUser.setAge(18);
+        testUser.setPassword("CorrectPassword");
```

Comment:
you can set all field for `testUser` in `@BeforeEach`and change some of them for every test to avoid duplicated code